Source/WebCore/dom/ViewTransition.cpp

```diff
@@ -95,6 +95,9 @@ RefPtr<ViewTransition> ViewTransition::resolveInboundCrossDocumentViewTransition
     if (!inboundViewTransitionParams)
         return nullptr;
 
+    if (MonotonicTime::now() - inboundViewTransitionParams->startTime > 4_s)
```

Comment:
We should probably share the `4_s` as a constant with the other timeout (`m_updateCallbackTimeout`).